bigbluebutton-html5/imports/ui/components/nav-bar/settings-dropdown/component.jsx

```diff
@@ -209,10 +228,12 @@ class SettingsDropdown extends PureComponent {
       shortcuts: OPEN_OPTIONS_AK,
     } = this.props;
 
+    const { isSettingOpen } = this.state;
+
     return (
       <Dropdown
         autoFocus
-        isOpen={this.state.isSettingOpen}
+        isOpen={isSettingOpen}
```

Comment:
I think this prop was renamed.
https://github.com/bigbluebutton/bigbluebutton/commit/8564e53ffb164ca991032ea4e6c03d8300faf50a#diff-5f1507eaefcaf30f75ce7be57afbf439R104

Reply:
Could you please confirm that it has to be `isOpen` now?